tutorials/sigfox-gw/index.md

```diff
@@ -193,21 +192,21 @@ The integration uses the Sigfox API V2 to automate configuring the callbacks. In
 
         (venv) $ cd ..
 
-1. Generate a Python client library from the Sigfox API V2 definition:
+1. Check that you have docker running, by executing for example:
 
-        curl -X POST -H "content-type:application/json" -d '{"swaggerUrl":"https://support.sigfox.com/api/apidocs"}' https://generator.swagger.io/api/gen/clients/python
+        (venv) $ docker ps
```

Comment:
Then what should I check for in the output?

Also, consider the shell alternative: command -v docker